src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Route, Switch, BrowserRouter as Router } from 'react-router-dom';
 import './App.css';
-import ProductDetails from './Pages/ProductDetails';
+import ProductDetails from './Pages/Product/ProductDetails/ProductDetails';
```

Comment:
it is so "long import",  propose 2 option:
1. Move to `ProductDetails` to  `./components/ProductDetails` and export it from `./components/index.ts`
2. Move to `ProductDetails` to  `./Pages/Product/components` and export it from `./components/index.ts`

I like 1 option. What you think?

Reply:
Fixed with 1st one. Also refactored old components import/export to index.ts in /Components forlder